service/src/main/java/bio/terra/landingzone/stairway/flight/ProtectedDataStepsDefinitionProvider.java

```diff
@@ -66,7 +66,7 @@ public List<Pair<Step, RetryRule>> get(
                 resourceNameProvider,
                 new AlertRulesHelper(armManagers.securityInsightsManager()),
                 landingZoneProtectedDataConfiguration),
-            RetryRules.cloud()));
+            RetryRules.cloudLongRunning()));
```

Comment:
I caught another flake on an IT run this morning. Taking one more stab at giving these more time and then will reassess.